Src/Fare.IntegrationTests/XegerTests.cs

```diff
@@ -121,9 +161,72 @@ public void GeneratedTextIsCorrectWithRexEngine(string pattern)
             @"\Sabc\S{3}111",
             @"^\S\S  (\S)+$",
             @"\\abc\\d",
-            @"\w+1\w{4}",
+            //@"\w+1\w{4}",
             @"\W+1\w?2\W{4}",
             @"^[^$]$"
         };
+        
+        public static TheoryData<string, string> RegexPatternCharset => new TheoryData<string, string>
+        {
+            {"[ab]{4,6}","ab"},
+            {"[ab]{4,6}c","abc"},
+            {"(a|b)*ab","ab"},
+            {"[A-Za-z0-9]","ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789"},
+            {"[A-Za-z0-9_]","ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789_"},
+            {"[A-Za-z]","ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz"},
+            {"[ \t]"," \t"},
+            {@"[(?<=\W)(?=\w)|(?<=\w)(?=\W)]",null},
+            {"[\x00-\x1F\x7F]",null},
+            {"[0-9]","0123456789"},
+            {"[^0-9]",null},
+            {"[\x21-\x7E]",null},
+            {"[a-z]","abcdefghijklmnopqrstuvwxyz"},
+            {"[\x20-\x7E]",null},
+            {"[ \t\r\n\v\f]"," \t\r\n\v\f"},
+            {"[^ \t\r\n\v\f]",null},
+            {"[A-Z]","ABCDEFGHIJKLMNOPQRSTUVWXYZ"},
+            {"[A-Fa-f0-9]","ABCDEFabcdef0123456789"},
+            {"in[du]","indu"},
+            {"x[0-9A-Z]","x0123456789ABCDEFGHIJKLMNOPQRSTUVWXYZ"},
+            {"[^A-M]in",null},
+            {".gr",null},
+            //@"\(.*l", - on "classic" went sometimes overflow (even before my changes)
```

Comment:
Based on this commented-out case, it looks like this data theory list is a duplicate of the existing one(?) If so, why?